stable/database/files/nuobackup

```diff
@@ -123,22 +123,19 @@ if [ "$backup_type" = "full" ]; then
       current=0
    fi
 
-   backupset=$( date "+%Y%m%dT%H%M%S" )
-
    # bump latest, and wrap around (simple ring-buffer)
    next=$(( (current + 1) % $NUODB_MAX_BACKUP_HISTORY ))
 else
    if [ -z "$latest" ]; then
       echo >&2 "No full backup has completed yet - try again later"
       exit -1
    fi
+fi
 
+if [ "$backup_type" = "report-latest" ]; then
    # retrieve the latest backupset name
    backupset=$(nuocmd --api-server $NUOCMD_API_SERVER get value --key $NUODB_BACKUP_KEY/$db_name/${backup_group}/${latest} )
    echo >&2 "Looking up $NUODB_BACKUP_KEY/$db_name/${backup_group}/${latest} returned: $backupset"
```

Comment:
I know you didn't write this code, but I'm not sure why this is going to stderr. This is just informational.

Reply:
My best guess is that this code path is used to obtain the latest successful full backupset in a sub-shell which will prevent the logging to be interpreted as an output:
```
latest=$(nuobackup --type report-latest --db-name demo --group cluster0)
```